torch/csrc/distributed/autograd/utils.cpp

```diff
@@ -62,12 +62,15 @@ DistAutogradContext* addRecvRpcBackward(
   return &autogradContext;
 }
 
-Message getMessageWithAutogradCheck(
+Message getMessageWithAutograd(
     const rpc::worker_id_t dstId,
     torch::distributed::rpc::Message&& wrappedRpcMsg,
     MessageType msgType) {
```

Comment:
Why do we pass msgType here? We just need to directly set `FORWARD_AUTOGRAD_REQ` on line 87

Reply:
because it could be passed with FORWARD_AUTOGRAD_REQ or FORWARD_AUTOGRAD_RSP